packages/algoliasearch/src/builds/browserLite.ts

```diff
@@ -26,6 +26,40 @@ import { createUserAgent, RequestOptions } from '@algolia/transporter';
 
 import { AlgoliaSearchOptions } from '../types';
 
+export {
```

Comment:
If eventually, a new modification introduces a new type that is not exported, api extractor will complain right?

Reply:
Exactly :slightly_smiling_face: